### ISSUE TITLE
Restoring Notepad++ from system tray.

### ISSUE THREAD

**Issue by systemmonkey42** [notepad-plus-plus/notepad-plus-plus#14009]
I use many keyboard shortcuts to minimize my reliance on the mouse (for various reasons), and I cannot find a way to
restore Notepad++ from the tray without the mouse.

Even launching ia new instance from the start menu (or pinned task bar icon) does nothing. 

Right clicking a file and selecting "Open with notepad++" (Which is Shift-F10 followed by "N") works, but if I want to return to an open document after notepad++ minimises to the tray, it seems an impossible task without the mouse.

What I would like to see:

When launching notepad++, if there is an existing instance running, and it is hidden in the tray, it should restore the existing window and make it active.

**Comment by victorel-petrovich** [comment]
> When launching notepad++, if there is an existing instance running, and it is hidden in the tray, it should restore the existing window and make it active.

I does restore for me, on: 
Notepad++ v8.5.4   (32-bit)
Build time : Jun 17 2023 - 20:40:14
Path : C:\Users\Vic\AppData\npp.8.5.4.portable\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : ON
Cloud Config : OFF
OS Name : Windows 7 Ultimate (32-bit)
OS Build : 7601.24546
Current ANSI codepage : 1252
Plugins : 
    mimeTools (2.9)
    NppConverter (4.5)
    NppExec (0.8.4)
    NppExport (0.4)
    PythonScript (2)

**Comment by systemmonkey42** [comment]
Thanks for the tip.  Here is the info

```text
Notepad++ v8.4.9   (64-bit)
Build time : Jan 27 2023 - 03:11:16
Path : D:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : P:\Notepad++\Navtrix-PC1
OS Name : Windows 10 Pro (64-bit) 
OS Version : 21H2
OS Build : 19044.3086
Current ANSI codepage : 1252
Plugins : 
    AutoSave (1.6.1)
    JSMinNPP (1.2205)
    JsonTools (3.2)
    mimeTools (2.9)
    NppConverter (4.5)
    NppExport (0.4)
    NppXmlTreeviewPlugin (2)
    PoorMansTSqlFormatterNppPlugin (1.6.13.31508)
    XMLTools (3.1.1.13)
```
I'll update to 8.5.x and re-test.  I've avoided 8.5 due to issues with the start menu, but the problem isn't new.

**Comment by systemmonkey42** [comment]
I installed 8.5.6, and it suddenly worked.  I rolled back
and tested various versions.  It appears to have been fixed in 8.5.0.  

Luckily I've also found a workaround for my menu issue, so I'll b able to keep using the latest.

Thanks

**Comment by victorel-petrovich** [comment]
You're welcome. Glad it works.
